ProjectHeads/Head.WinAppSdk.props

```diff
@@ -1,10 +1,7 @@
 <Project>
-  <Import Project="$(MSBuildThisFileDirectory)\..\MultiTarget\PackageReferences\WinAppSdk.props" />
-  <Import Project="$(MSBuildThisFileDirectory)\..\MultiTarget\WinUI.Extra.props" />
-
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>$(WinAppSdkTargetFramework)</TargetFramework>
+    <TargetFramework>$(WinAppSdkTargetFramework.Split(';')[0])</TargetFramework>
```

Comment:
Small thing to discuss figure out later, maybe open an issue for when we add .net8 later once that ships. We probably want our Sample apps running against the latest vs. the oldest version? So, either we should grab the last one if we're always going to order them the same or should maybe have a separate property we can set for this elsewhere?

Reply:
Grabbing the last index was a bit more complicated, I figured we stick with LTS for now. Can adjust or reorder them later if needed.